diff --git a/node_modules/smee-client/index.js b/node_modules/smee-client/index.js
index 496c560..c803462 100644
--- a/node_modules/smee-client/index.js
+++ b/node_modules/smee-client/index.js
@@ -26,12 +26,14 @@ class Client {
 
     const req = superagent.post(target).send(data.body)
 
+    const data_len = JSON.stringify(data.body).length
     delete data.body
 
     Object.keys(data).forEach(key => {
       req.set(key, data[key])
     })
 
+    req.set('content-length', `${data_len}`)
     req.end((err, res) => {
       if (err) {
         this.logger.error(err)
